### ISSUE TITLE
Remind @someone to do something

### ISSUE THREAD

**Issue by seveneightn9ne** [seveneightn9ne/keybase-reminder-bot#8]
Potential for abuse? Maybe only allow it if you're a channel with both of them?

**Comment by svoop** [comment]
Good idea, however, how about always mentioning the user who has created the reminder without any changes to the interface?
